manifests/install.pp

```diff
@@ -5,8 +5,14 @@
 
   if $qpid::ensure == 'absent' {
     $_package_ensure = 'purged'
+    $_tools_ensure   = $qpid::ensure
   } else {
     $_package_ensure = $qpid::version
+    $_tools_ensure   = 'present'
+  }
+
+  class { 'qpid::tools':
+    ensure => $_tools_ensure,
```

Comment:
I think you could just pass `$qpid::ensure` directly here.

Reply:
:+1: